.golangci-kal.yml

```
@@ -30,6 +30,7 @@ linters:
               - "ssatags" # Ensure array fields have the appropriate listType markers
               - "statusoptional" # Ensure all first children within status should be optional.
               - "statussubresource" # All root objects that have a `status` field should have a status subresource.
+              - "notimestamp" # Prevents usage of 'TimeStamp' fields
```

```suggestion
              - "notimestamp" # Prevents usage of 'Timestamp' fields
```
Based on the earlier feedback https://github.com/kubernetes-sigs/kube-api-linter/pull/105#discussion_r2222599855, If relevant here as well.